src/containers/my-cooperations/cooperation-notes/create-or-edit-note/CreateOrEditNote.tsx

```diff
@@ -47,20 +44,23 @@ const CreateOrEditNote = ({
 
   const { userId, userRole } = useAppSelector((state) => state.appMain)
 
-  const getUserData: () => Promise<AxiosResponse<UserResponse>> = useCallback(
-    () => userService.getUserById(userId, userRole as UserRole),
+  const getUserData: () => Promise<UserResponse> = useCallback(
```

Comment:
You can remove type from here, and also, please, add braces to arrow function if it doesn't fit into single line
```suggestion
  const getUserData = useCallback(
```